File: PE-2.go
package main

import "fmt"

func main() {
	fib1, fib2, sum := 0, 1, 0
	
	for fib1 < 4000000 {
		temp := fib1
		fib1 = fib2
		fib2 += temp
		if fib1 % 2 == 0 {
			sum += fib1
		}
	}

	fmt.Println(sum)
}

File: PE-2.py
# https://projecteuler.net/problem=2

fib1 = 0
fib2 = 1
_sum = 0

while fib1 < 4000000:
    fib1, fib2 = fib2, fib1 + fib2
    if fib1 % 2 == 0:
        _sum += fib1

print(_sum)
